### PR TITLE
Added a new contact UI

### LINKED ISSUE #5: Create a new contact UI

### PR DESCRIPTION
**Pull request by mr-jestin-roy**
Fixes #5 

Screen Shot 2022-09-14 at 3 42 14 PM

Screen Shot 2022-09-14 at 3 33 23 PM

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Edit.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Pane/Form";
+
+const EditNotePane = ({ fetchNotes, showPane, setShowPane, note }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Edit.jsx:7]: Why is `EditNotePane` kept under `Dashboard/Contacts/Edit.jsx`?

File: app/javascript/src/components/Dashboard/Contacts/Create.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import ContactForm from "./Pane/Form";
+
+const ContactPane = ({ showPane, onClose }) => (
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Create.jsx:7]: This should be named `Create` itself and import it using custom alias wherever required. The filename and component name should match. Also prefixing with the `Contact` keyword is unnecessary over here. Refer: https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Avoid%20Unnecessary%20Namespacing

**Review by yedhink — COMMENTED** [review]
@mr-jestin-roy

File: app/javascript/src/components/Dashboard/Contacts/Edit.jsx (hunk 1 above)
Comment (mr-jestin-roy) [reply app/javascript/src/components/Dashboard/Contacts/Edit.jsx:7]: Sorry, it's pretty redundant. I used the same format from NotePane for "Add new note". I'll remove it before the next PR.

**Pull request merged**